dart/lib/io.dart

```diff
@@ -2,6 +2,5 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-export 'src/base.dart';
-export 'src/browser.dart';
+export 'src/io_client.dart';
```

Comment:
@rxlabz do those file naming changes cause any breaking changes on the client's app? do they need to fix imports or something else?

Reply:
it has no impact unless you use a package internal path which is bad :)